Jenkinsfile

```diff
@@ -5,6 +5,5 @@ buildNPM {
   runSonarqube = false
   runScripts = [
    ['test:core':'--karma.singleRun --karma.browsers ChromeDocker --karma.reporters mocha junit --coverage'],
```

Comment:
This can just be reduced to `test` now and `test:core` can be removed from `package.json` since they do the same thing.

Reply:
Thanks for catching this @ryandberger!